examples/fastly/fastly.toml

```
@@ -0,0 +1,23 @@
+# This file describes a Fastly Compute@Edge package. To learn more visit:
+# https://developer.fastly.com/reference/fastly-toml/
+
+language = "javascript"
+manifest_version = 2
+name = "fastly planetscale f1"
+service_id = ""
+
+[local_server.backends.psdb]
+url = "https://aws-eu-west-2.connect.psdb.cloud"
```

```suggestion
url = "https://aws.connect.psdb.cloud"
```

You'd want to use the global endpoint here.